docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -1 +1,98 @@
 = Glossary of terms related to {ProductName}
+:icons: font
+:source-highlighter: highlightjs
+
+**{ProductName}** +
+A platform to automate the process of building, testing, and releasing applications. {ProductName} offers enterprise-grade security and customizable feature sets.   
+
+**build pipeline customization** +
+The ability to update and manage build pipelines for each component in an application. 
+
+**cluster** +
+A Kubernetes deployment with nodes that run containerized applications and a control plane that manages the nodes.
+
+**commit** +
+A change to one or more files. In {ProductName}, commits that you make to a linked git repository move through the pipeline and automatically get published.
```

Comment:
This is specific to content tracked in source control. These are not just any changes, but just those that have been added to the tree. 

https://git-scm.com/docs/git-commit
https://www.atlassian.com/git/tutorials/saving-changes/git-commit

Reply:
How about just deleting this definition? I don't think it's necessary for the kind of users that we'll have.

Reply:
that works for me.